src/pages/Dongwu/Login/LoginArticle/LoginForm/LoginInput/LoginInput.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styles from './LoginInput.module.scss';
+
+class LoginInput extends React.Component {
+  // 수정사항 있는듯 한데 어떻게 변경해야 할지 감이 오지 않습니다.
+  loginValidation = () => {
+    const { state, setParentState } = this.props;
+    state[`loginId`].includes(`@`) && state[`loginPw`].length > 8
+      ? setParentState(`disabled`, false)
+      : setParentState(`disabled`, true);
+  };
```

Comment:
```suggestion
  loginValidation = () => {
    const { loginId, loginPw, toggleBtnDisabled } = this.props
    const isFormValid = loginId.includes('@') && loginPw.length > 8
    
    toggleBtnDisabled(isFormValid)
  };
```
위와 같은 형태로 코드 작성할 수 있도록 해주세요
- 객체에 접근할 때 dot notation 필요한 경우와 그렇지 않은 경우에 대해서 생각해보시고 그 내용을 댓글로 달아주세요
- 따옴표 사용과 백틱 사용 역시 마찬가지입니다
- state 자체를 props로 받아오게 되면 작성해주신 것과 같이 해당 컴포넌트의 상태인지 위에서 받아온 상태인지 알기가 어렵습니다

Reply:
종택님! 리뷰 정말 감사합니다!

우선은 변경사항인 module.scss와 백틱 수정 외에는 전부 진행했습니다.

먼저, 리뷰해주신 사항 중 dot notation에 대한 내용 공부하고 댓글 남깁니다.

- 제 생각에 dot notation은 객체 내 key에 직접적인 접근을 할 때 사용하는 방식인 것 같습니다.
 따라서, 해당 코드 내에서는 dot notation을 활용한 접근이 올바른 방법인 것 같아 수정했습니다.
 
 - 반대로 backet notation은 특정 객체를 목표로 하는 것이 아닌, 임의의 여러 객체에서 사용할 수 있는 추상적인 함수나 메서드 선언에 사용되는 것 같습니다.
 
 따라서 해당 블럭의 코드는 dot notation으로 변경하였습니다.
 
 반대로 bracket notation을 활용할 수 있는 부분이 있는 것 같아 LoginForm.js의 setLoginInfo 메서드는 brack notation으로 선언하여 전달했습니다.
 
 ```javascript
   setLoginInfo = (labelId, value) => {
    this.setState(prevState => {
      return { ...prevState, [labelId]: value };
    });
  };
 ```
 위 코드가 LoginForm.js 의 setLoginInfo 메서드입니다.
 
- 백틱과 따옴표의 사용은 Js 코드가 내부에 존재하느냐, 하지 않느냐로 구분할 수 있는 것 같습니다.

- 문제는 해당 프로젝트 코드 내에서 알고 있음에도 무분별하게 사용하고 있었던 점인데, 앞으로는 보다 신경쓸 수 있도록 하겠습니다.
 
 잘못 생각하고 있는 부분 피드백해주시면 정말 감사하겠습니다.